Biosample landing page

mlacadie opened the pull request:
Add biosample landing page and gwas landing page home screen button
fix #154 
fix #157 
fix #162

mlacadie committed:
biosample home layout

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
start page

mlacadie committed:
better

mlacadie committed:
types

mlacadie committed:
start mouse

mlacadie committed:
config fix

mlacadie committed:
more mouse lines

mlacadie committed:
Update mouseMapping.ts

mlacadie committed:
Update mouseMapping.ts

mlacadie committed:
insets

mlacadie committed:
onclick accordian

mlacadie committed:
styling

mlacadie committed:
auto scroll and new mapping names

mlacadie committed:
added gwas card

mlacadie committed:
hide body maps

mlacadie committed:
small changes

mlacadie committed:
small change

mlacadie committed:
thyroid and tooltip

mlacadie committed:
disabling

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
fix #162

mlacadie committed:
fix #157

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
disabling

mlacadie committed:
Update ExploreMore.tsx

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

jpfisher72 reviewed (commented):
Nice job with this!

Some small differences I found between this and encode:
<img width="1434" height="1011" alt="Screenshot 2025-11-12 at 1 53 43 PM" src="https://github.com/user-attachments/assets/548db730-55b9-4209-9e72-db31a4756db4" />

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
fix insetsvgs and epiglottis

mlacadie committed:
fix link

mlacadie committed:
inset fix

mlacadie committed:
Update InsetSvgs.tsx

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
clipping from encode

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
Merge branch 'main' into biosampleLandingPage

mlacadie committed:
merge fixes

jpfisher72 committed:
Phylogenetic Tree/Sequence Alignment Plot on cCRE/conservation, PhastCons + PhyloP (#282)

Adds 241 Mammals Sequence Coverage on cCRE/conservation under new tab (Only works for chr22 right now since that's the only data we have)

Switches cCRE/conservation table to displaying PhastCons and PhyloP

JaiiR320 committed:
WGBS + TF Peak Tracks (#283)

* added support for wgbs and tf motif tracks

* fixed chromhmm tooltip

* fix tf track rehydration

* regenerated gql types

* bump package versions

* ignore devserve and opencode config folders

* add comment for context

* remove devserve artifacts

* Delete custom.md

jpfisher72 committed:
ui-components v2 (#284)

* Change all imports

* Change to notFound() to fix .well-known issue

* Stable references to slots

* Bump viz to 1.3.1 to fix key error

* fix AssayTable/GeneExpressionTable bad setState

* Fix BiosampleTable event subscription

NishiPhalke committed:
Added Overlapping Representative ChIP-seq Peaks in TF Motif tab of cCRE entity page and added ucsc gb logo in entity headers (#285)

* added overlapping rpeaks, open in ucsc browser button

* added representative peaks with motifs on cCRE entity page

* tab title changes

* added TF Motif Icon

* build fix

* removed unused imports

jpfisher72 committed:
Refactor table plot sync + other misc cleanup + MOHD ATAC & RNA seq (#290)

mlacadie committed:
delete

mlacadie committed:
remove

mlacadie committed:
Merge branch 'staging' into biosampleLandingPage

mlacadie committed:
table working

mlacadie committed:
Update BiosampleLandingHeader.tsx

mlacadie committed:
Update layout.tsx
